api/api/accounts/views.py

```diff
@@ -20,3 +23,14 @@ class RetrieveUpdateUserView(generics.RetrieveUpdateAPIView):
     permission_classes = (IsOwnerOrAdmin,)
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+
+class ListUserRoles(generics.ListAPIView):
```

Comment:
Correct me if I am wrong. This endpoint returns roles of the users where they are core members of a club.
What's stopping us from making it a generic endpoint that returns all the user roles?

Moreover if we are not using user-id from url path, it doesn't make sense to have url like: `/accounts/<int:pk>/roles/`, let's just have it like: `/accounts/me/roles/`

Reply:
@Aksh-Bansal-dev this works?

Reply:
I think `/accounts/me/roles/` would be fine. We can just get the user-id from the token.

Reply:
Not about the url ... Returning all roles is okay or is this specific to roles like (co,co-co,and core) as mentioned in the issue #198

Reply:
Yeah, returning all roles would be better.